content/en/opentelemetry/guide/otel_api_tracing_interoperability.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Interoperability of OpenTelemetry API and Datadog instrumented traces
+kind: guide
+further_reading:
+- link: "/tracing/trace_collection/otel_instrumentation/"
+  tag: "Documentation"
+  text: "Custom Instrumentation with OpenTelemetry API"
+- link: "/tracing/trace_collection/trace_context_propagation/"
+  tag: "Documentation"
+  text: "Trace Context Propagation in Datadog"
+
+---
+
+## Overview
+
+Datadog tracing libraries provide an implementation of the [OpenTelemetry API][1] for instrumenting your code. This means you can maintain vendor-neutral instrumentation of all your services, while still taking advantage of Datadog's native implementation, features, and products. 
```

Comment:
One small detail that I'd like for the documentation to call out is that the user would replace the OpenTelemetry SDK with the Datadog tracing library in the instrumented application. I'm not sure what the wording would look like though

Reply:
Good idea. With my next commit, I took the last bulled from the list below, and changed it a bit to include this piece of info.

Reply:
Looks good!